app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :books
+  has_many :reviews
```

Comment:
Always add destroy dependencies kub
```suggestion
  has_many :books, dependent: :destroy
  has_many :reviews, dependent: :destroy
```